specs/swagger.yml

```diff
@@ -4337,6 +4337,9 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/entityRef'
+      config:
+        type: object
```

Comment:
map definitions (i.e. config stuff) should be

```
  config:
    type: object
    additionalProperties:
      type: object    
```

this translates to map[string]interface{} in Go.